
# SyntaxError:

# print 'hello world!'


# NameError:

# print(age)


# IndexError:

# numbers = [1, 2, 3, 4, 5]
# numbers[5]


# ModuleNotFoundError:

# import maths


# AttributeError:

# import math
# math.PI 


# KeyError:

# users = {'name': 'Asab', 'age': 250, 'country': 'Finland'}
# users['county']


# TypeError:

# 4 + '3'


# ImportError:

# from math import power


# ValueError:

# int('12a')


# ZeroDivisionError:

# 1/0
